src/test/java/io/confluent/connect/hdfs/avro/DataWriterAvroTest.java

```diff
@@ -326,7 +331,8 @@ public void testRebalance() throws Exception {
     }
 
     hdfsWriter.write(sinkRecords);
-    hdfsWriter.close();
+    hdfsWriter.close(assignment);
```

Comment:
Shouldn't this `close(newAssignment)` since that was the last assignment opened?

This actually seems pretty concerning -- why isn't `close`-ing the wrong assignment causing an error.